.github/workflows/main.yml

```diff
@@ -432,6 +432,19 @@ jobs:
         run: |
           RUST_TEST_THREADS=1 cargo test --manifest-path="libvcx/Cargo.toml" -F "pool_tests";
 
+  test-integration-resolver:
```

Comment:
Following up the testing re-approach, integration tests are now typically have `ignore` annotation and we typically run unit and integration tests separately. While the resolves resides under aries-vcx, we should probably stay consistent